app/controllers/api/teams.php

```diff
@@ -427,8 +427,8 @@
         $response->dynamic(
             $membership
             ->setAttribute('teamName', $team->getAttribute('name'))
-            ->setAttribute('userName', $user->getAttribute('name'))
-            ->setAttribute('userEmail', $user->getAttribute('email')),
+            ->setAttribute('userName', $name)
+            ->setAttribute('userEmail', $email),
```

Comment:
```suggestion
            ->setAttribute('userName', $invitee->getAttribute('name'))
            ->setAttribute('userEmail', $invitee->getAttribute('email')),
```

It achieves the same thing, but it can be a better idea to fetch name and email stored in the database than use the one passed in the request

Reply:
Yes, it makes sense, to be able to get the accurate data if the user is already exist, changes are committed.